src/clj/game/cards/ice.clj

```diff
@@ -1171,6 +1171,10 @@
                            :delayed-completion true
                            :effect (effect (tag-runner :runner eid 1))
                            :msg "give the Runner 1 tag"}}]}
+                           
+    "Self-Adapting Code Wall"
+    {:subroutines [end-the-run]
+        :flags {:cannot-lower-strength true}}
 
```

Comment:
Justify this so that the colons preceding `subroutines` and `flags` are vertically aligned.